libcxx/test/libcxx/atomics/atomics.syn/compatible_with_stdatomic.compile.pass.cpp

```diff
@@ -7,7 +7,10 @@
 //===----------------------------------------------------------------------===//
 
 // UNSUPPORTED: no-threads
-// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// On Android, libc++'s <stdatomic.h> header always redirects to <atomic>, even before C++23.
+// UNSUPPORTED: c++03
```

Comment:
why is c++03 on a separate line now?

Reply:
It probably shouldn't be on a separate line -- it looks like libc++ does support `<atomic>` for C++03.

Though now I'm wondering if the `UNSUPPORTED` directive ought to be `XFAIL` instead. I think the libc++ project usually prefers `XFAIL` when we know that a test should fail.

Reply:
I think I'd move back c++03 on the same line as the other ones. I'm neutral on XFAIL vs UNSUPPORTED -- as you mention we already have a dedicated test for the inverse behavior.